### PR TITLE
[INLONG-7389][Manager][Sort] Add audit id info for source

### LINKED ISSUE #7389: [Improve][Manager][Sort] Add audit id info for source
### Description

Add audit id info for source.

### InLong Component

InLong Manager, InLong Sort

### Are you willing to submit PR?

- [X] Yes, I am willing to submit a PR!

### Code of Conduct

- [X] I agree to follow this project's [Code of Conduct](https://www.apache.org/foundation/policies/conduct)

### PR DESCRIPTION
**Pull request by fuweng11**
### Prepare a Pull Request

- Fixes #7389 

### Motivation

Add audit id info for source.

### Modifications

Add audit id info for source.

### CODE REVIEW

File: inlong-sort/sort-common/src/test/java/org/apache/inlong/sort/protocol/node/extract/KafkaExtractNodeTest.java [hunk 1]
```diff
@@ -49,7 +51,12 @@ public KafkaExtractNode getTestObject() {
         List<FieldInfo> fields = Arrays.asList(
                 new FieldInfo("name", new StringFormatInfo()),
                 new FieldInfo("age", new IntFormatInfo()));
-        return new KafkaExtractNode("1", "kafka_input", fields, null, null, "workerCsv",
+        Map<String, String> properties = new HashMap<>();
+        List<String> auditIds = new ArrayList<>();
+        auditIds.add("7");
```
Comment (vernedeng) [inline inlong-sort/sort-common/src/test/java/org/apache/inlong/sort/protocol/node/extract/KafkaExtractNodeTest.java:56]: Metric number, make it constant plz

**Review by vernedeng — CHANGES_REQUESTED** [review]

**Review by vernedeng — APPROVED** [review]
lgtm

File: inlong-sort/sort-common/src/test/java/org/apache/inlong/sort/protocol/node/extract/KafkaExtractNodeTest.java (hunk 1 above)
Comment (healchow) [reply inlong-sort/sort-common/src/test/java/org/apache/inlong/sort/protocol/node/extract/KafkaExtractNodeTest.java:56]: It looks like this magic number is still there, and it is recommended to extract it as a constant in this class, with an explanation of what it means.

File: inlong-manager/manager-service/src/main/java/org/apache/inlong/manager/service/resource/sort/DefaultSortConfigOperator.java [hunk 2]
```diff
@@ -121,6 +124,15 @@ private GroupInfo getGroupInfo(InlongGroupInfo groupInfo, List<InlongStreamInfo>
             // build a stream info from the nodes and relations
             List<StreamSource> sources = sourceMap.get(streamId);
             List<StreamSink> sinks = sinkMap.get(streamId);
+            // get audit list by sink type
+            List<String> auditIds = new ArrayList<>();
+            for (StreamSink sink : sinks) {
+                auditIds.add(auditService.getAuditId(sink.getSinkType(), false));
```
Comment (gong) [inline inlong-manager/manager-service/src/main/java/org/apache/inlong/manager/service/resource/sort/DefaultSortConfigOperator.java:130]: Would it throw exception if I don't use audit service?

**Comment by EMsnap** [comment]
pls remove the audit parameter in inlongmsg

**Review by EMsnap — APPROVED** [review]

**Review by dockerzhang — APPROVED** [review]

**Pull request merged**